src/main/java/net/earthcomputer/clientcommands/features/VillagerRngSimulator.java

```diff
@@ -333,7 +413,7 @@ public long[] crackSeed() {
                 }
                 rand.nextInt(100);
             }
-            if (rand.nextInt(1000) >= ticksBetweenSounds) {
+            if (rand.nextInt(1000) >= ticksBetweenSounds - 80 - 1) {
```

Comment:
did you test that this works?

Reply:
Yes, it also makes much more sense with the comparisons the loop is doing

Reply:
ok